apps/web/modules/survey/editor/components/logic-editor-actions.tsx

```
@@ -98,153 +100,179 @@ export function LogicEditorActions({
-                      handleValuesChange(idx, {
-                        variableId: val,
-                        value: {
-                          type: "static",
-                          value: "",
-                        },
-                      });
-                    }}
-                    comboboxClasses="grow"
-                    emptyDropdownText={t("environments.surveys.edit.add_a_variable_to_calculate")}
-                  />
+          <div className="ml-5 flex items-center gap-x-2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)